luadox/render/html.py

```diff
@@ -285,6 +290,8 @@ def _render_html(self, topref: TopRef, lines: List[str]) -> Generator[
 
         css = self.config.get('project', 'css', fallback=None)
         if css:
+            # The stylesheet is always copied to doc root, so take only the filename
+            _, css = os.path.split(css)
```

Comment:
This commit slipped in from the other PR.  Ideally PRs would be independent from one another, which requires a separate branch per PR on your fork.  This allows PRs to be merged independent of one another.  It's not a big deal in this one, because I already merged #17 so this will be ignored, but it's more of a problem for the other PRs.